### DISCUSSION TITLE
How to detect Bundle Reboot Pending in WiX v4

### DISCUSSION THREAD

**Discussion by kalyani1217** [wixtoolset/issues#8012] [Questions]
The ResumeType.RebootPending, which is present in v3, is now not available in v4.  We would like to know if any bundle reboot pendings are present when I'm trying to launch the setup. I have used the engine variable "RebootPending" to find the system reboot pendings. How do I know if any bundle reboot pendings are present?

As a workaround, I have added a custom action that writes a volatile key to the registry if the ScheduleReboot action gets triggered. And I'm checking whether this volatile key is present or not when I'm launching/modifying the setup. If a volatile key is present, then I'm asking the user to restart the machine.

Please suggest if any better approaches are there.

**Comment by barnson** [comment]
https://github.com/wixtoolset/issues/issues/6668

**Comment by kalyani1217** [comment]
Hi, @barnson , Sorry to say this. But I didn't understand how to find out if any bundle reboots were pending.
